src/sass/components/_card.scss

```diff
@@ -161,18 +161,31 @@
     }
 
     &__candidate-read-more-link {
-      left: 0;
       margin-left: 10px;
       top: 0;
-      font-weight: lighter;
+      font-family: $body-font-stack;
+      font-size: 14px;
+      font-weight: 100 !important;
```

Comment:
Hi @ixley can we discuss when we are looking at screens together? I would like these "learn more" links to be extremely light to minimize visual clutter.